playbook/app/pb_kits/playbook/pb_advanced_table/docs/_advanced_table_selectable_rows_no_subrows_rails.md

```diff
@@ -1 +1,5 @@
-`selectable_rows` can also be used with tables without nested row data. Set `enable_toggle_expansion: none` to adjust the Advanced Table styling to fit a table without subrows.
\ No newline at end of file
+`selectable_rows` can also be used with tables without nested row data. Set `enable_toggle_expansion: none` to adjust the Advanced Table styling to fit a table without subrows.
+
+The data attribute `data-selected-rows` on the parent advanced table div will update dynamically with each selection to show an array of ids corresponding to the current selection. 
+
+__NOTE__: Each object within the `table_data` Array must contain a unique id in order to attach an id to all Rows for this to function. 
```

Comment:
Just curious on what the standard is for the way we write our kits with uppercase and lowercase. I see on line 1 its "Advanced Table" then on line 3 its "advanced table".

Reply:
The standard should be upper case when referring to a kit, but I think for line 3 I need to make it pb_advanced_table div for clarity